### PR TITLE
feat(pci.storage.databases): change unit for network metrics from octects to megaoctects

### PR DESCRIPTION
**Pull request by jperchoc**
DATATR-29

Signed-off-by: Jonathan Perchoc <jonathan.perchoc@ovhcloud.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #DATATR-29
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [ ] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

Convert network metrics from octects to megaoctects.

### :flags: Translations

- [x] TRANS-44559

### CODE REVIEW

**Review by ghyenne — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/metrics/metrics.controller.js [hunk 1]
```diff
@@ -106,8 +106,19 @@ export default class MetricsCtrl {
...
       ).then((data) => {
-        const metrics = sortBy(data.metrics, 'hostname');
-        this.metricsData[data.name].data = data;
+        const metricData = data;
+        if (['net_send', 'net_receive'].includes(availableMetric)) {
+          metricData.metrics[0].dataPoints = data.metrics[0].dataPoints.map(
+            (point) => {
+              return {
+                timestamp: point.timestamp,
+                value: point.value / 1000000,
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/storages/databases/database/metrics/metrics.controller.js:115]: **suggestion:** Better to add it as a constant with a meaningful name which will result in better understanding.

**Review by anooparveti — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/metrics/metrics.controller.js (hunk 1 above)
Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/storages/databases/database/metrics/metrics.controller.js]: I have updated the code to use constants, it looks cleaner :smile: https://github.com/ovh/manager/pull/7124/commits/b0992dc00c31840deffebd72d3358801e92a6bbb

**Review by anooparveti — DISMISSED** [review]

**Review by Dimfacion — APPROVED** [review]

**Comment by antleblanc** [comment]
Pulling translations.

**Comment by antleblanc** [comment]
@jperchoc Could you please resolve existing conflicts? (appears after pulling translations).
Thank you!

**Review by anooparveti — DISMISSED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**